### PR TITLE
Instigator

### PR DESCRIPTION
**Pull request by lars-wu**
InstigatorInfoLong:
As the Instigator, it's your job to turn the crewmates against each other. Each time a Crewmate is voted out in a meeting, as long as you are alive, an additional Crewmate who voted for the innocent player will die after the meeting. The number of additional players dying is determined by the host

### CODE REVIEW

**Comment by 0xDrMoe** [comment]
This has been converted to Draft as we are not doing role pushes until next full release

File: Patches/ExilePatch.cs [hunk 1]
```diff
@@ -1,11 +1,10 @@
+using System.Linq;
 using AmongUs.Data;
 using HarmonyLib;
-using System.Linq;
```
Comment (0xDrMoe) [inline Patches/ExilePatch.cs:3]: why did this get removed

File: Resources/Lang/en_US.json [hunk 2]
```diff
@@ -540,6 +541,7 @@
   "BerserkerInfo": "Kill to increase your level",
   "EvilMiniInfo": "No one can hurt you until you grow up",
   "BlackmailerInfo": "Silence other players",
+  "InstigatorInfo": "Sow discord among the crewmates",
```
Comment (0xDrMoe) [inline Resources/Lang/en_US.json:544]: Sow? I'm so lost loool

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]

File: Patches/ExilePatch.cs (hunk 1 above)
Comment (Tommy-XL) [reply Patches/ExilePatch.cs:3]: Maybe it's just not used in the code

Comment (Gurge44) [reply Patches/ExilePatch.cs:3]: When you do an auto code cleanup in VS, it removes unnecessary using directives

Comment (lars-wu) [reply Patches/ExilePatch.cs:3]: What @Gurge44 said.

File: Resources/Lang/en_US.json (hunk 2 above)
Comment (lars-wu) [reply Resources/Lang/en_US.json:544]: Is it not correct english? haha. How would you say it? Translator says it's fine.

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:544]: sow is correct, but simply do "Cause discord among the crewmates" imo

File: Patches/ExilePatch.cs [hunk 3]
```diff
@@ -214,7 +217,8 @@ CustomRoles.Tracefinder or
                 CustomRoles.Doctor or
                 CustomRoles.Alchemist or
                 CustomRoles.Bomber or
-                CustomRoles.Undertaker
+                CustomRoles.Undertaker or
+                CustomRoles.Instigator
                 ) pc.RpcResetAbilityCooldown();
```
Comment (Tommy-XL) [inline Patches/ExilePatch.cs:223]: I'll temporarily remove this code because I've re coded it to #206

**Review by Tommy-XL — APPROVED** [review]

**Comment by Tommy-XL** [comment]
Need fix conflict

**Pull request merged**